draft-ietf-tls-tls13.md

```diff
@@ -4000,8 +4000,13 @@ by IANA
   "Recommended": ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384,
   rsa_pss_sha256, rsa_pss_sha384, rsa_pss_sha512, ed25519.
 
-Finally, this document obsoletes a registry originally created in {{RFC5246}}.
-IANA [shall/has] updated it to list values 7-223 as "Reserved".
+  [[TODO: How to denote that values ending in 0x00 through 0x03 are also reserved?]]
```

Comment:
We already have a "Values with the first byte..." sentence in its definition. Just add a second:

"Values with the second byte in the range 0-3 that are not currently allocated are reserved for backwards compatibility."